README.md

```diff
@@ -1,39 +1,5 @@
-# README
+# Decidim Bulletin Board
 
-The Bulletin Board is a service composed by an Encryption Engine and an Append-Only Log. External queries to the Encryption Engine are allowed through an API.
+This repository follows the monorepo pattern and includes the following projects:
 
-* Ruby version: 2.6
-* Rails version: 6
-
-* System dependencies:
-
-The Bulletin Board depends on the existence of a Decidim installation.
-
-
-* Installation:
-
-First, clone the repository and enter in the new folder:
-```
-git clone git@github.com:decidim/decidim-bulletin-board.git
-cd decidim-bulletin-board
-```
-
-Now, execute these commands:
-```
-bundle install
-yarn install
-rails db:create
-rails db:migrate
-```
-
-
-* How to run the rubocop linter
-```
-bundle exec rubocop
-```
-
-
-* How to run the test suite
-```
-bundle exec rspec
-```
+- **decidim-bulletin-board-app**: A Ruby on Rails application that contains the Bulletin Board service.
```

Comment:
maybe we can link the app readme from here, what do you think?

Reply:
Yeah! I think it will be a great addition 😄

Reply:
@leio10 I added the link! It will work when it's merged to `master`. Can you double check please? Thanks!